workshop/serializers.py

```diff
@@ -526,7 +527,7 @@ def save(self, **kwargs):
     class Meta:
         model = Workshop
         fields = (
-            'id', 'title', 'description', 'date', 'time', 'location', 'latitude',
+            'id', 'title', 'description', 'is_workshop', 'date', 'time', 'location', 'latitude',
```

Comment:
Same here for entities, we can directly set `is_workshop=False` in workshop create. No need to take this field from the user.